src/redux/reducer.js

```diff
@@ -160,7 +161,26 @@ export const reducer = createReducer(initialState, (builder) => {
     });
 
     builder.addCase(TREE_DATA, (state, action) => {
-        state.treeData = action.treeData;
+        // filtering non DIRECTORY elements from action.treeData
```

Comment:
Add a TODO and some explanation because thoss filterings will become useless when reducer pass to typescript and when `action.treeData.rootDirectories` and `action.treeData.mapData` will be typed to `IDirectory[]` and `Record<string, IDirectory>` respectively